operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -229,19 +229,33 @@ private String getRuntimeEncryptionSecret() {
     return getDomain().getRuntimeEncryptionSecret();
   }
 
+  private boolean isSipProtocol(NetworkAccessPoint nap) {
+    return "sip".equals(nap.getProtocol()) || "sips".equals(nap.getProtocol());
+  }
+
   private List<V1ContainerPort> getContainerPorts() {
     if (scan != null) {
 
       List<V1ContainerPort> ports = new ArrayList<>();
       if (scan.getNetworkAccessPoints() != null) {
         for (NetworkAccessPoint nap : scan.getNetworkAccessPoints()) {
           String napName = nap.getName();
+
```

Comment:
This method is starting to get large. Might be a good idea to factor out the loop body. The two cases of adding a port seem similar enough that common code could be extracted from them.

Reply:
done